docs/application/native/guides/multimedia/pose-detection.md

```diff
@@ -0,0 +1,342 @@
+
+![Body pose](./media/mediavision_pose_public_model_def.png),
+
+while `-1` means that there is no landmarks, you can create a mapping file. Suppose that the file is `pose_mapping.txt`, then, you can create `pose_mapping.txt` as
+
+```
+1
+2
+-1
+3
+4
+5
+6
+7
+8
+-1
+9
+10
+11
+12
+13
+14
+```
+
+`1` denotes that the first landmark of the model corresponds to `MV_INFERENCE_HUMAN_POSE_HEAD`.  `-1` at the 3rd denotes that there is no corresponding landmark `MV_INFERENCE_HUMAN_POSE_THORAX`. `3` at the 4th denotes that the 3rd landmark of the model corresponds to `MV_INFERENCE_HUMAN_POSE_RIGHT_SHOULDER`. You can know how it works with following table.
+
+**Table: Example of how  [public pose model](https://github.com/tyoungroy/PoseEstimationForMobile) to map the definition**
+
+| Value | Defintion | pose_mapping.txt |
+| - | - | - |
+| 1 | MV_INFERENCE_HUMAN_POSE_HEAD | 1 |
+| 2 | MV_INFERENCE_HUMAN_POSE_NECK | 2 |
+| 3 | MV_INFERENCE_HUMAN_POSE_THORAX | -1 |
+| 4 | MV_INFERENCE_HUMAN_POSE_RIGHT_SHOULDER | 3 |
+| 5 | MV_INFERENCE_HUMAN_POSE_RIGHT_ELBOW | 4 |
+| 6 | MV_INFERENCE_HUMAN_POSE_RIGHT_WRIST | 5 |
+| 7 | MV_INFERENCE_HUMAN_POSE_LEFT_SHOULDER | 6 |
+| 8 | MV_INFERENCE_HUMAN_POSE_LEFT_ELBOW | 7 |
+| 9 | MV_INFERENCE_HUMAN_POSE_LEFT_WRIST | 8 |
+| 10 | MV_INFERENCE_HUMAN_POSE_PELVIS | -1 |
+| 11 | MV_INFERENCE_HUMAN_POSE_RIGHT_HIP | 9 |
+| 12 | MV_INFERENCE_HUMAN_POSE_RIGHT_KNEE | 10 |
+| 13 | MV_INFERENCE_HUMAN_POSE_RIGHT_ANKLE | 11 |
+| 14 | MV_INFERENCE_HUMAN_POSE_LEFT_HIP | 12 |
+| 15 | MV_INFERENCE_HUMAN_POSE_LEFT_KNEE | 13 |
+| 16 | MV_INFERENCE_HUMAN_POSE_LEFT_ANKLE | 14 |
+
+---
+
+Motion capture file (MoCap) includes the movements of objects or people. There are various MoCap formats, but well-known BVH (BioVision Hierarchy) file is supported in Media Vision. BVH file has Hierarchy structure to provide landmark information and the structure can be changed. It means that landmarks information is differenct from the definition. To use them correctly, you have to map those information to landmarks based on the definition. For example, an [example](file://./media/mediavision_pose_bvh_sample.bvh) BVH file describes a squat pose as,
+
+![Body pose](./media/mediavision_pose_bvh_sample.png)
+, with 15 landmarks. It starts with Hips and ends with RightRoot. Thus, you can create a mapping file, named of `mocap_mapping.txt` as
+
+```
+Hips,10
+Neck,2
+Head,1
+LeftUpArm,7
+LeftLowArm,8
+LeftHand,9
+RightUpArm,4
+RightLowArm,5
+RightHand,6
+LeftUpLeg,14
+LeftLowLeg,15
+LeftFoot,16
+RightUpLeg,11
+RightLowLeg,12
+RightFoot,13
+```
```

Comment:
As your comment there is wrong description. 
I fixed `It starts with hips and ends right foot` to `it starts with hips and ends with left foot`.

Also, 
I revised ` BVH file has Hierarchy structure to provide landmark information the structure can be changed.` with `BVH file has Hierarchy structure to provide landmark information with landmarks' name and the structure can be changed`.

So, `Hips, 10` means that the landmark named of `Hips` in the bvh file corresponds to 10th definition `MV_INFERENCE_HUMAN_POSE_PELVIS`. The order in the mapping file is not important because that it can be mapped by the name.

If there is no mapped landmark, you don't need to list it. For example, index 3 is missing. It means that, in this bvh file, the 3rd definition, `MV_INFERENCE_HUMAN_POSE_THORAX` , is not defined and not used.